### PR TITLE
docs: typo

### PR DESCRIPTION
**Pull request by alfiedotwtf**
Wrong verb.

### CODE REVIEW

File: src/doc/reference.md [hunk 1]
```diff
@@ -3402,7 +3402,7 @@ Used inside an array pattern, `..` stands for any number of elements, when the
 `advanced_slice_patterns` feature gate is turned on. This wildcard can be used
 at most once for a given array, which implies that it cannot be used to
 specifically match elements that are at an unknown distance from both ends of a
-array, like `[.., 42, ..]`. If followed by a variable name, it will bind the
+array, like `[.., 42, ..]`. If proceeded by a variable name, it will bind the
```
Comment (steveklabnik) [inline src/doc/reference.md]: it's 'preceed' not 'proceed`. The variable comes before.

**Comment by alfie** [comment]
Gah! That's what I meant to type. I guess I need spellcheck in vim. Redoing.

**Comment by steveklabnik** [comment]
Hehe it's all good :heart: Thanks!

**Pull request merged**